packages/table/src/tableBodyCells.tsx

```diff
@@ -187,7 +193,8 @@ export class TableBodyCells extends React.Component<ITableBodyCellsProps, {}> {
         const cellLoading = baseCell.props.loading != null ? baseCell.props.loading : loading;
 
         const style = { ...baseCell.props.style, ...Rect.style(rect) };
-        return React.cloneElement(baseCell, { className, key, loading: cellLoading, style });
+        const isFocused = focusedCell != null && focusedCell.row === rowIndex && focusedCell.col === columnIndex;
```

Comment:
Can we add a `CellUtils.cellsEqual` utility function? (in `common/internal/cellUtils` + a few quick unit tests). This check seems like something that could pop up again.

Reply:
Don't we have something like this for regions?

Reply:
We do.

Reply:
I'm... not sure how cellsEqual or anything of the like would make sense here. I'm comparing a focused coordinates with two numbers passed into the function. If I made a cellsEqual, what would I be passing it as arguments, exactly?

Reply:
Ah, didn't look closely at the comparison values. You're right.